conseil-common/src/main/scala/tech/cryptonomic/conseil/common/tezos/SqlOperations.scala

```diff
@@ -0,0 +1,85 @@
+package tech.cryptonomic.conseil.common.tezos
+
+import slick.jdbc.PostgresProfile.api._
+import tech.cryptonomic.conseil.common.generic.chain.{DataOperations, MetadataOperations}
+import tech.cryptonomic.conseil.common.generic.chain.DataTypes.{
+  Field,
+  FormattedField,
+  Predicate,
+  Query,
+  QueryResponse,
+  SimpleField
+}
+import tech.cryptonomic.conseil.common.util.DatabaseUtil
+
+import scala.concurrent.{ExecutionContext, Future}
+
+object SqlOperations {
```

Comment:
We'd need to add comments here for completeness, even a reminder will do for now, for our team to fill later